ui/lib/apps/InstanceProfiling/index.tsx

```diff
@@ -1,24 +1,26 @@
 import React from 'react'
-import { HashRouter as Router, Route, Routes } from 'react-router-dom'
+import {
+  HashRouter as Router,
+  Route,
+  Routes,
+  useParams,
+} from 'react-router-dom'
 
 import { Root } from '@lib/components'
+import { Detail, List } from './pages'
 
-import DetailPage from './pages/Detail'
-import ListPage from './pages/List'
+function DetailPageWrapper() {
```

Comment:
Maybe this function can be a common pattern :) I believe for all pages using `useClientRequest` we'd better use a similar solution.

Reply:
Make sense, let me try it.